dart/lib/src/sentry_options.dart

```diff
@@ -279,6 +279,9 @@ class SentryOptions {
   /// If enabled, [scopeObservers] will be called when mutating scope.
   bool enableScopeSync = true;
 
+  /// Sets the ProGuard uuid for Android platform.
+  String? proGuardUuid;
```

Comment:
```suggestion
  /// Sets the ProGuard uuid for Android platform.
  String? proguardUuid;
```

Let's align on the naming.

Reply:
done! thanks!